.gitignore

```diff
@@ -25,3 +25,8 @@ yarn-error.log*
 
 /.idea
 .vscode
+
+jspm_packages
+
+# Serverless directories
+.serverless
```

Comment:
that's redundant, as it is already there by default (line#13)

Reply:
Many thanks)